server/server.go

```diff
@@ -1118,6 +1124,39 @@ func (s *Server) GetRegions() []*core.RegionInfo {
 	return nil
 }
 
+// GetServiceLabels returns ApiAccessPaths by given service label
+func (s *Server) GetServiceLabels(serviceLabel string) []apiutil.APIAccessPath {
+	if apis, ok := s.serviceLabels[serviceLabel]; ok {
+		return apis
+	}
+	return nil
+}
+
+// GetAPIAccessServiceLabel returns service label by given access path
+func (s *Server) GetAPIAccessServiceLabel(accessPath apiutil.APIAccessPath) string {
```

Comment:
It seems these methods are only used in `test`, so would we use it in the main logic in the future? If yes, how about add a `TODO` comment? If no, may be add `test only` related comment will be better?

Reply:
updated